src/main/resources/styles/reporting.css

```diff
@@ -1,5 +1,9 @@
 <style>
 
+h2 {
+	font-size: 24px !important;
```

Comment:
If so just leave it. Some day we hopefully delete this legacy code but it won't be easy....